src/libs/Clipboard/index.js

```diff
@@ -62,7 +62,9 @@ function setHTMLSync(html, text) {
 
     if (isComposer) {
         firstAnchorChild.setSelectionRange(originalSelection.start, originalSelection.end, originalSelection.direction);
-    } else {
+    } else if (originalSelection.anchorNode && originalSelection.focusNode) {
+        // When copying to clipboard here, since there will be no user selection the original values of anchorNode and focusNode will be null
```

Comment:
@AmjedNazzal Thanks for the update! Unfortunately, we have a language issue here I think. I'm not sure what whs the intention here. This can be probably improved easily; I use ChatGPT for things like this